department-of-veterans-affairs/va.gov-cms
# VACMS-3872: Content publisher can not VBO modify field values.

## Description

See #3872. 

## Testing done

Manual and phpunit

## Screenshots

![Facility_Status___VA_gov_CMS_and_Facility_Status___VA_gov_CMS](https://user-images.githubusercontent.com/643678/103300001-e25f0980-49cb-11eb-9d39-c163224207ee.jpg)

## QA steps

As user sandy.vadeo@va.gov with content_publisher role
1. Go to admin/content/facility-status 
2. You should not see any actions


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 

i let @davidconlon and @mmiddaugh know about this change in https://dsva.slack.com/archives/CDHBKAL9W/p1609261279295200?thread_ts=1608677600.276600&cid=CDHBKAL9W ... 

No other existing users of this dashboard are affected.
